snowflake/DataIngestWithSnowpipe.txt

```diff
@@ -0,0 +1,43 @@
+-- Select Database -- 
+
+use database TEST_DB;
+
+--Create integration --
+
+create notification integration snowpipe_event_integration
+    ENABLED=TRUE
+    TYPE=QUEUE
+    NOTIFICATION_PROVIDER=AZURE_STORAGE_QUEUE
+    AZURE_STORAGE_QUEUE_PRIMARY_URI='https://<storage-account>.queue.core.windows.net/<queue>' 
+    AZURE_TENANT_ID='<tenant-id>';
+    
+desc notification integration snowpipe_event_integration;
+ 
+-- Create stage -- 
+
+create or replace stage azure_stage
+  URL='azure://<storage-account>.blob.core.windows.net/<blob>'
+  CREDENTIALS=(AZURE_SAS_TOKEN='<sas-token>');
+  
+-- Create snowpipe -- 
+ 
+create pipe TEST_DB.PUBLIC.testsnowpipe
+    AUTO_INGEST=TRUE
+    INTEGRATION=snowpipe_event_integration
+    as
+    copy into TEST_TABLE_SNOWPIPE
+    from @TEST_DB.PUBLIC.azure_stage
+    FILE_FORMAT=(TYPE='CSV');
+    
+ select SYSTEM$PIPE_STATUS(TEST_DB.PUBLIC.testsnowpipe);
+ 
+ -- Show results --
+ 
+ select * from TEST_TABLE_SNOWPIPE;
+ 
+ -- Drop objects --
+
+ drop notification integration if exists snowpipe_event_integration;
+ drop stage if exists azure_stage;
+ drop pipe if exists TEST_DB.PUBLIC.testsnowpipe;
+ drop table if exists TEST_DB.PUBLIC.TEST_TABLE_SNOWPIPE;
```

Comment:
Let's use `use schema` in the beginning and we can have: `drop table if exists TEST_TABLE_SNOWPIPE;`